source/settings/settings-reference.txt

```diff
@@ -158,10 +158,10 @@ You can configure the following settings on the |compass| interface:
      - Remain logged in when using the :manual:`MONGODB-OIDC </core/security-oidc/>` 
        authentication mechanism. 
 
-   * - Use Generative AI
```

Comment:
[question] "Enable AI features" is listed twice in the table. Is that expected?

Reply:
This moved to the artificial intelligence tab and should be listed once.